zeppelin-plugins/notebookrepo/mongo/src/test/java/org/apache/zeppelin/notebook/repo/MongoNotebookRepoTest.java

```diff
@@ -49,7 +49,9 @@
   @Before
   public void setUp() throws IOException {
     String bindIp = "localhost";
-    int port = new ServerSocket(0).getLocalPort();
+    ServerSocket socket = new ServerSocket(0);
+    int port = socket.getLocalPort();
+    socket.close();
```

Comment:
I think socket should be closed before the free port is used, otherwise mongodb will not start because the port is occupied. 
Please correct me if I say anything wrong. Thanks @zjffdu